fe/src/main/java/org/apache/doris/load/routineload/KafkaRoutineLoadJob.java

```diff
@@ -128,6 +132,9 @@ private void convertCustomProperties() throws DdlException {
                 convertedCustomProperties.put(entry.getKey(), entry.getValue());
             }
         }
+        if (convertedCustomProperties.containsKey(KAFKA_DEFAULT_OFFSETS)) {
+            kafkaDefaultOffSet = convertedCustomProperties.get(KAFKA_DEFAULT_OFFSETS);
```

Comment:
```suggestion
            kafkaDefaultOffSet = convertedCustomProperties.remove(KAFKA_DEFAULT_OFFSETS);
```

convertedCustomProperties 中只能包含可以被 librdkafka 识别的properties，所以这个default offset 需要单独移除